### PR TITLE
Adding the task placement constraint in the main.tf for ecs service creation

### PR DESCRIPTION
**Pull request by balajimoses**
Adding the task placement constraint in the main.tf for ecs service creation

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -68,7 +68,8 @@ module "service" {
 
   assign_public_ip = var.assign_public_ip
 
-  enable_execute_command = var.enable_execute_command
+  enable_execute_command     = var.enable_execute_command
+  task_placement_constraints = each.value.task_placement_constraints
```
Comment (niroz89) [inline main.tf:72]: @balajimoses is this a separate variable or part of `service_map` ?

Comment (smoneyan) [reply main.tf:72]: yea.. shouldn't this be from `var.task_placement_constraints` ?

Comment (balajimoses) [reply main.tf:72]: its a separate variables , but its not called anywhere in the main.tf. So i thought to use inside the service map. Please suggest your opinion on this

**Review by smoneyan — APPROVED** [review]
Looks good.

**Pull request merged**